src/App.tsx

```diff
@@ -1,35 +1,21 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.scss'
+import { Navigate, Route, Routes } from "react-router-dom";
```

Comment:
Az importoknál csináljunk egy hierarchiát, a rendszeresség és olvashatóság miatt.
```
import { Navigate, Route, Routes } from "react-router-dom";

import AppNavbar from "./components/AppNavbar";
import Movies from "./pages/Movies";
import Sandbox from "./pages/Sandbox";

import "./App.scss";
```
Felül a package importok, utána komponens library (react bootstrap, mui) utána saját importok, utána css importok. A többihez nem fogom odaírni, de csináljuk meg azoknál is.